compiler/packages/react-forgive/client/src/extension.ts

```diff
@@ -0,0 +1,62 @@
+import * as path from 'path';
+import {ExtensionContext, window as Window} from 'vscode';
+
+import {
+  LanguageClient,
+  LanguageClientOptions,
+  ServerOptions,
+  TransportKind,
+} from 'vscode-languageclient/node';
+
+let client: LanguageClient;
+
+export function activate(context: ExtensionContext) {
+  const serverModule = context.asAbsolutePath(path.join('dist', 'server.js'));
+
+  // If the extension is launched in debug mode then the debug server options are used
+  // Otherwise the run options are used
+  const serverOptions: ServerOptions = {
+    run: {
+      module: serverModule,
+      transport: TransportKind.ipc,
+      options: {cwd: process.cwd()},
```

Comment:
I just stumbled upon this PR by accident, but I can tell you from experience that you shouldn’t rely on `process.cwd()` in a VSCode extension or language server. It may vary based on which OS the user uses and how they opened their editor. It’ll cause all sorts of unexpected issues.